### ISSUE TITLE
Proposal antenna for rover 

### ISSUE THREAD

**Issue by kikislater** [Francklin2/RTKLIB_Touchscreen_GUI#19]
Hi Francklin, 

I found this antenna which seems to be the same as the implemented ebee one or heliceo or drobit system => 25g 

![Github Logo](http://gpsworld.com/wp-content/uploads/2016/03/HX-CH6601A-e1458679320729.png)

> Antenna Spec
> Frequency Range   GPS  L1/L2
> GLONASS  L1/L2
> BDS B1/B2
> Impedance     50ohm
> Polarization  RHCP
> Axial Ratio   ≤3dB
> Azimuth Coverage  360°
> Output VSWR   ≤2.0
> Peak Gain     2dBi
> LNA Spec
> LNA Gain  33±2dB
> Noise Figure  ≤2dB
> Output VSWR   ≤2.0
> Operation Voltage     3~12VDC
> Operation Current     ≤55mA
> Ripple    ±2dB
> Mechanical Spec
> Dimension     ¢27.5*59mm
> Weight    ≤25g
> Connector     SMA
> Environment Spec
> Operating Temp    -40℃～+70℃
> Storage Temp  -40℃～+70℃
> Humidity  95% No-condensing

http://gpsworld.com/harxon-offers-new-gnss-helix-antenna/

http://www.harxon.com/products-detail.php?ProId=83

**Comment by kikislater** [comment]
An idea about price is ~250€ without freight cost

**Comment by Francklin2** [comment]
A bit expensive but it will be a nice one for the rover (25grams) or a good alternative for the talysman, we can add it to the shopping list of the readme

**Comment by Francklin2** [comment]
Around the same price I saw this new one at Csgshop, not really a antenna because it got the intégrated M8T , it could be a middle price solution for the rover side (there is cheaper ones aroud 90$ at Drotek or Csgshop)
http://www.csgshop.com/product.php?id_product=222

http://www.csgshop.com/img/p/222-518-thickbox.jpg

**Comment by kikislater** [comment]
With this size and weight of 25g, there are some different and a little bit cheaper for L1 ~around 180$

Model   Frequeny
HX-CH4601A  GPS L1/L2，GLONASS L1/L2
HX-CH6601A  GPS L1/L2，GLONASS L1/L2 BDS B1/B2
HX-CH3602A、HX-CH3603A、HX-CH3604A    GPS L1、BDS B1、GLONASS L1

Of course your M8T you list is cheaper but only L1

**Comment by Francklin2** [comment]
We can propose a list of compatible GNSS and antenna in different prices range, I only search for the low cost side GNSS for the moment (Ublox,Skytrack or NVS) 
http://navspark.mybigcommerce.com/ns-raw-carrier-phase-raw-measurement-output-gps-receiver/
http://www.navspark.mybigcommerce.com/ns-hp-rtk-capable-gnss-receiver/

**Comment by kikislater** [comment]
According to ublox specs we could have 10Hz with setting only GPS or GLONASS in M8T 
P7/35 => 
https://www.u-blox.com/sites/default/files/NEO-LEA-M8T-FW3_DataSheet_(UBX-15025193).pdf

May be usefull for rover on a drone

**Comment by Francklin2** [comment]
Yes the M8T can run at 10hz, you have to set it in Ucenter, to change the refresh rate in hz, you have to change the timing in ms to 200ms instead of 500ms., over 5hz some values become red but it works i will add how to setup the M8T in the wiki

**Comment by kikislater** [comment]
Before known test, I prefer using 10Hz on a drone : more reliable

**Comment by kikislater** [comment]
Francklin, did you see the latest version : 
https://pixhawk.org/dev/rtk-basestation-touch-tft

**Comment by Francklin2** [comment]
Yes I saw it, I have build the first version without screen and we started our project  in the same time, we can tell him that our project is released too., I didnt try his software yet because It need the correct touchscreen . I will try to download it to see how he extract the info from rtkrcv.
For the rover part I noticed that the Drotel Smartnav is not too expensive (288€)
